### ISSUE TITLE
Hyperlactatorachia

### ISSUE THREAD

**Issue by AliceChoury** [obophenotype/human-phenotype-ontology#4342]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Hyperlactatorachia

### 2. Synonyms
 Hyperlacticacidemia in LCR

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

High concentration of lactate in LCR

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
[Lactic acidosis]( https://hpo.jax.org/app/browse/term/HP:0003128)

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

https://link.springer.com/article/10.1007/s13341-011-0038-z

**Comment by nicolevasilevsky** [comment]
what is LCR?

**Comment by pnrobinson** [comment]
are we talking about lactate clearance rate? This is not a synonym.

**Comment by pnrobinson** [comment]
This is a synoynm of https://hpo.jax.org/app/browse/term/HP:0002490
We should add "Hyperlactatorachia" as a synoynm.
I will also see if we can add a comment about the LCR

**Comment by pnrobinson** [comment]
I have added Hypolactatorachia and Hyperlactatorachia as synonyms to the corresponding terms.
